app/components/about_page/belief_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="sm:grid sm:grid-cols-12 sm:gap-x-6 pb-14">
   <div class="flex sm:col-span-2 justify-center ">
-    <%= image_tag @belief[:image_path], alt: @belief[:image_alt], class: 'w-20 h-20' %>
+    <%= image_tag @belief[:image_path], alt: '', class: 'w-20 h-20' %>
```

Comment:
Nit: Now that we're not using the `image_alts` anymore 💪.  Can we delete those fields from the [locale](https://github.com/TheOdinProject/theodinproject/blob/main/config/locales/en.yml#L35) file they are stored in please?